docs/csharp/tutorials/microservices.md

```diff
@@ -63,7 +63,9 @@ referenced above contains sections to each of the available installations.
 
 You'll also need to install a number of command line tools that support
 ASP.NET core development. The command line templates use Yeoman, Bower,
-Grunt, and Gulp. You may already have many of these tools, but if not,
+Grunt, and Gulp to be installed using the node.js's package manager npm which 
+can be installed from [nodejs.org](https://nodejs.org). 
```

Comment:
I find the new wording awkward. Maybe move the mention of npm before the other tools?